pkg/app/server/grpcapi/api.go

```diff
@@ -533,6 +537,20 @@ func (a *API) GetPlanPreviewResults(ctx context.Context, req *apiservice.GetPlan
 		if cmd.Type != model.Command_BUILD_PLAN_PREVIEW {
 			return nil, status.Error(codes.FailedPrecondition, fmt.Sprintf("Command %s is not a plan preview command", commandID))
 		}
+		pipedStatus, err := getPipedStatus(a.pipedStatCache, cmd.PipedId)
```

Comment:
nit: add an empty line after a return block

```suggestion

		pipedStatus, err := getPipedStatus(a.pipedStatCache, cmd.PipedId)
```